sources/settings-migrations/v1.51.0/kubernetes-additional-settings/src/main.rs

```diff
@@ -0,0 +1,23 @@
+use migration_helpers::common_migrations::AddSettingsMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+// We added new kubernetes settings to configure:
+// - min/max duration before an unused image is garbage-collected
+// - max number of image pulls in parallel
+// - mapping length of UIDs and GIDs
+fn run() -> Result<()> {
+    migrate(AddSettingsMigration(&[
+        "settings.kubernetes.image-minimum-gc-age",
+        "settings.kubernetes.image-maximum-gc-age",
+        "settings.kubernetes.max-parallel-image-pulls",
+        "settings.kubernetes.ids-per-pod",
+    ]))
+}
+
```

Comment:
Nit: Usually we add:

```suggestion
// Returning a Result from main makes it print a Debug representation of the error, but with Snafu
// we have nice Display representations of the error, so we wrap "main" (run) and print any error.
// https://github.com/shepmaster/snafu/issues/110
```

Reply:
Yeah I have included those comments for migrations in the past, but I purposefully left them here since it's just repeated info. LMK if you prefer we keep them for each migration file